audit/projects/k8s-infra-public-pii/description.json

```diff
@@ -0,0 +1,11 @@
+{
+  "createTime": "2021-05-12T09:38:46.426Z",
+  "lifecycleState": "ACTIVE",
+  "name": "k8s-infra-public-pii",
+  "parent": {
+    "id": "758905017065",
+    "type": "organization"
+  },
+  "projectId": "k8s-infra-public-pii",
+  "projectNumber": "226195303281"
+}
```

Comment:
Is there a PR open for this? @ameukam

Reply:
Yes, @ameukam reviewed before I finished my review. https://github.com/kubernetes/k8s.io/pull/2031